docs/third-party-developers/extensibility/checkout-block/available-filters.md

```diff
@@ -290,23 +290,38 @@ registerCheckoutFilters( 'automatic-coupon-extension', {
 ### Prevent a snackbar notice from appearing for coupons
 
 If you want to prevent a coupon apply notice from appearing, you can use the `showApplyCouponNotice` filter. If it returns `false` then the notice will not be created.
+```ts
```

Comment:
```suggestion

```ts
```

Reply:
Our MD linter detected the missing line break before the code fence block in https://github.com/woocommerce/woocommerce-blocks/actions/runs/6180445606/job/16776936842?pr=10941:

<img width="883" alt="Screenshot 2023-09-14 at 10 23 30" src="https://github.com/woocommerce/woocommerce-blocks/assets/3323310/8ddf8e38-5a5a-4145-be3a-951de342a083">